.code-samples.meilisearch.yaml

```diff
@@ -199,6 +199,14 @@ search_parameter_guide_offset_1: |-
   await client.index('movies').search('shifu', offset: 1);
 search_parameter_guide_limit_1: |-
   await client.index('movies').search('shifu', limit: 2);
+search_parameter_guide_matching_strategy_1: |-
+  import 'package:meilisearch/src/matching_strategy_enum.dart';
+
+  await client.index('movies').search('big fat liar', matchingStrategy: MatchingStrategy.last);
```

Comment:
I updated the code samples to match the [actual implementation](https://github.com/meilisearch/meilisearch-dart/pull/216) that uses `enum`  😉